### DISCUSSION TITLE
[Solved] Players' profiles are opened only after page reload

### DISCUSSION THREAD

**Discussion by SelenIT** [cssbattle/forum#218] [Bugs reports]
When I click a player's name (including mine) to see their stats, first I get an empty dark page. Only when I reload it, the player's data are displayed correctly. On the first click, no errors or warnings appear in the console, but the 2nd network request after a long (about ~30s) timeout displays a "Failed to load response data" message in the Network devtools panel:
<img width="1383" alt="Screenshot 2022-01-23 at 23 29 53" src="https://user-images.githubusercontent.com/282754/150696913-d231b134-e4a3-4480-826a-4f8173cc626f.png">
It's not a critical issue, but could you please look into it?

**Comment by chinchang** [comment]
Hi @SelenIT Can you please tell where did you open this player page from? As in homepage leaderboard or somewhere else?

**Reply by SelenIT** [reply]
Hi @chinchang thank you! Yes, I tried to open links from the homepage leaderboard in the right column. As far as I can tell, the same issue happens with all 10 links there, and it reproduces in different browsers on different computers 😳

**Reply by chinchang** [reply]
Thats really strange, because navigation from homepage to player profile is already a real reload, so it should work fine....as it does for me. Just a safe check, can you do a hard refresh on the homepage once and then try? Lemme know.

**Reply by SelenIT** [reply]
I tried to record what happens when I click:

https://user-images.githubusercontent.com/282754/150748412-6da38d49-1df1-4ccd-96cb-b19a0a8bce3e.mp4

Apparently there is one long request there, but I'm not sure whether it's the cause of the behavior I see. At about 1:38, I clicked the browser's Reload button.

**Reply by Beowolve** [reply]
I can confirm that behavior, its exactly the same for me.

**Reply by chinchang** [reply]
ahh..now I see the difference. You were clicking player name and i was trying with the avatar. Avatar works but the player name does not. Fixing...

**Reply by chinchang** [reply]
THis is fixed now! Thanks for reporting @SelenIT :)

**Reply by SelenIT** [reply]
Thank you!
